bin/rucio-admin

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-# Copyright 2012-2020 CERN
+# Copyright 2012-2021 CERN
```

Comment:
why was the `# -*- coding: utf-8 -*-` part removed?

Reply:
I ran the `tools/add_header` script with the `-i` flag (`tools/add_header -i bin/rucio-admin`) and it deleted that line

Reply:
Related with #4756

Reply:
Thanks, that's the missing bit :)